jetty-websocket/websocket-core/src/main/java/org/eclipse/jetty/websocket/core/io/WebSocketCoreConnection.java

```diff
@@ -163,6 +163,11 @@ public void setSession(WebSocketCoreSession session)
         this.incomingFrames = session;
     }
 
+    public ExtensionStack getExtensionStack()
+    {
+        return extensionStack;
+    }
+
```

Comment:
Access to extension stack is NOT required and discouraged

Reply:
I needed that at one stage when I was passing just the connection to something that completed the response handshake.  I agree that access to the stack should not be necessary, but getting the negotiated extensions is a reasonable getter to have.  However I think I currently don't need either.